react-native-aztec/android/src/main/java/org/wordpress/mobile/ReactNativeAztec/ReactTextInputShadowNodeFork.java

```diff
@@ -0,0 +1,240 @@
+package org.wordpress.mobile.ReactNativeAztec;
+
+import android.os.Build;
+import android.text.Layout;
+import android.util.TypedValue;
+import android.view.ViewGroup;
+import android.widget.EditText;
+
+import androidx.annotation.Nullable;
+import androidx.annotation.VisibleForTesting;
+import androidx.core.view.ViewCompat;
+
+import com.facebook.infer.annotation.Assertions;
+import com.facebook.react.bridge.JSApplicationIllegalArgumentException;
+import com.facebook.react.bridge.ReadableMap;
+import com.facebook.react.uimanager.Spacing;
+import com.facebook.react.uimanager.ThemedReactContext;
+import com.facebook.react.uimanager.UIViewOperationQueue;
+import com.facebook.react.uimanager.annotations.ReactProp;
+import com.facebook.react.views.text.ReactBaseTextShadowNode;
+import com.facebook.react.views.text.ReactTextUpdate;
+import com.facebook.react.views.textinput.ReactTextInputLocalData;
+import com.facebook.react.views.view.MeasureUtil;
+import com.facebook.yoga.YogaMeasureFunction;
+import com.facebook.yoga.YogaMeasureMode;
+import com.facebook.yoga.YogaMeasureOutput;
+import com.facebook.yoga.YogaNode;
+
+/**
+ * This is a fork from {@link com.facebook.react.views.textinput.ReactTextInputShadowNode} for the purpose
+ * of customizing that class so that the construction of the dummy {@link EditText} instance
+ * can be overridden (see {@link ReactTextInputShadowNodeFork#createDummyEditText(ThemedReactContext)}).
+ */
+public class ReactTextInputShadowNodeFork extends ReactBaseTextShadowNode
```

Comment:
Is there some plan on how can we follow improvements in `ReactAztecTextShadowNode` or `ReactBaseTextShadowNode` which will be potentially presented in the new version of RN?

Reply:
Not really. Ideally, we'll be able to get React Native itself updated to allow overriding the construction of the `EditText` instance [like this](https://github.com/wordpress-mobile/gutenberg-mobile/pull/1560/commits/048f07042603cf9579098eb250989f18dd2592b4#diff-50887f88a59f4879f48ea11a48b84e14R74-R95). Otherwise (and until then) we will just have to manually pull in any updates.

Reply:
Got this fix [merged into React Native](https://github.com/facebook/react-native/commit/a5b5d1a805a9c54d325763b432be1cf2c8811dc9), so once we start using a version of React Native that includes that change, we won't need to have this `ReactTextInputShadowNodeFork` anymore! 😄